src/Microsoft.DotNet.Tools.Compiler/Program.cs

```diff
@@ -347,7 +349,8 @@ private static bool CompileProject(ProjectContext context, CompilerCommandApp ar
 
             if (success && !args.NoHostValue && compilationOptions.EmitEntryPoint.GetValueOrDefault())
             {
-                var projectContext = ProjectContext.Create(context.ProjectDirectory, context.TargetFramework, new[] { RuntimeIdentifier.Current });
+                var runnableRids = PlatformServices.Default.Runtime.GetAllCandidateRuntimeIdentifiers();
+                var projectContext = ProjectContext.Create(context.ProjectDirectory, context.TargetFramework, runnableRids);
```

Comment:
Why are we invoking these two lines again? They seem the exact same ones as the ones above (254 and 255). Does the output of these change because of something that happens in between these calls?